pipeline/splicing/calcVarPriors.py

```diff
@@ -595,27 +598,43 @@ def getVarLocation(variant, boundaries):
             return "UTR_variant"
         return "intron_variant"
 
-def getFastaSeq(chrom, rangeStart, rangeStop):
+def getFastaSeq(chrom, rangeStart, rangeStop, plusSeq=True):
```

Comment:
I changed it to `plusStrandSeq` to be more descriptive because it refers to whether the sequence on the plus strand is returned `plusStrandSeq=True` or the sequence on the minus strand is returned `plusStrandSeq=False`